Passkey: Adjust IPA passkey config error log level

justin-stephenson opened the pull request:
IPA passkey configuration may not be retrieved if IPA does not contain passkey support. Lower the error level of log messages associated with this failure.

alexey-tikhonov commented:
@justin-stephenson, what code handles those errors up the chain?

I'd like to make sure there is something like '(SSSDBG_IMPORTANT_INFO, "Passkey feature isn't configured at IPA server")' printed (ideally once per start).

justin-stephenson commented:
> @justin-stephenson, what code handles those errors up the chain?

Passkey configuration retrieval is part of a sequence of multiple different operations in [src/providers/ipa/ipa_subdomains.c](https://github.com/SSSD/sssd/blob/master/src/providers/ipa/ipa_subdomains.c#L2748), retrieving data like cert mapping rules, view names, etc.
> 
> I'd like to make sure there is something like '(SSSDBG_IMPORTANT_INFO, "Passkey feature isn't configured at IPA server")' printed (ideally once per start).

I added this message, please check.

alexey-tikhonov commented:
> > I'd like to make sure there is something like '(SSSDBG_IMPORTANT_INFO, "Passkey feature isn't configured at IPA server")' printed (ideally once per start).
> 
> I added this message, please check.

Thank you.

Did you also check if other users of `ipa_get_config_done()` log something on a default log level in case of fail (so we don't suppress it)?

justin-stephenson commented:
> > > I'd like to make sure there is something like '(SSSDBG_IMPORTANT_INFO, "Passkey feature isn't configured at IPA server")' printed (ideally once per start).
> > 
> > 
> > I added this message, please check.
> 
> Thank you.
> 
> Did you also check if other users of `ipa_get_config_done()` log something on a default log level in case of fail (so we don't suppress it)?

I added another commit to switch these other callers to SSSDBG_IMPORTANT_INFO if `ipa_get_config_send()` fails

alexey-tikhonov commented:
Thank you, ACK.

ikerexxe reviewed (approved):
LGTM. Thanks for the patches!

alexey-tikhonov commented:
Pushed PR: https://github.com/SSSD/sssd/pull/6751

* `master`
    * fa326be9cb29d97d6000e783b02656f60f7f8fb2 - IPA: Log missing IPA config data on default level
    * fe751c316c631240311fba62409f2b6a38be0d50 - Passkey: Adjust IPA passkey config error log level
* `sssd-2-9`
    * 16275d9b4697d6c2763818e25181f6243d697926 - IPA: Log missing IPA config data on default level
    * 270f0ba011de6265df925ebdb8ce489cba64cb60 - Passkey: Adjust IPA passkey config error log level
